docs/source/examples/interactive-development.rst

```diff
@@ -0,0 +1,170 @@
+Start a Development Cluster
+===========================
+
+SkyPilot makes the interactive development very easy on clouds. It helps you:
+
+#. Launch: a cluster with GPU or any specific requirement with a single command.
+#. Connect: Quickly connect to the cluster with the cluster name.
+#. Autostop: Automatically stop the cluster after a period of time for cost savings.
+
+Launch
+------
+
+To launch a cluster with a cheap GPU for developemnt:
+
+.. code-block:: bash
+
+  # Launch a cluster with 1 NVIDIA GPU and syncing the local working directory to the
+  # cluster.
+  sky launch -c dev --gpus T4 --workdir .
+
+
+.. note::
+
+  View the supported GPUs with the :code:`sky show-gpus` command.
+
+
+Autostop
+--------
+
+SkyPilot allows you to automatically stop the cluster after a period of time to save costs. You can set the autostop time with a single command.
+
+.. code-block:: bash
+
+  # Auto stop the cluster after 5 hours
+  sky autostop -i 300 dev
+
+Or add an additional flag :code:`-i` during the launch:
+
+.. code-block:: bash
+
+  # Launch a cluster with auto stop after 5 hours
+  sky launch -c dev --gpus T4 --workdir . -i 300
+
+For more details of auto stopping, check out: :ref:`auto-stop`. It helps you avoid idle machines from costing you a fortune, when you want a
+machine to stop automatically after you go to bed or forget to stop it.
+
+
+Connect
+-------
+
+A user can easily connect to the cluster with the following ways:
+
+SSH
+~~~
+
+SkyPilot will automatically configure the SSH setting for a user to connect to the cluster with the cluster name after it is launched.
```

Comment:
```suggestion
SkyPilot will automatically configure the SSH setting for a cluster, so that users can connect to the cluster with the cluster name:
```